src/__integrations__/pagopaApi.browser.integration.test.ts

```diff
@@ -0,0 +1,139 @@
+import { Server } from 'http';
```

Comment:
what is the purpose of these tests? why use a mock server?

Reply:
The purpose of the tests is to guarantee that the custom fetches work in a browser environment